assets/style.css

```diff
@@ -0,0 +1,998 @@
+/* General definitions */
+
+@font-face {
+  font-family: "Roboto Condensed";
+  src: url(fonts/RobotoCondensed-Regular.ttf);
+}
+
+html * {
+  font-family: "Roboto Condensed", arial, sans-serif;
+}
+
+body {
+  background-color: #f4f5f6;
+  min-width: 450px;
+}
+
+.container,
+.container-fluid {
+  max-width: 1310px;
+}
+
+.bg-primary {
+  background-color: #f72210 !important;
+}
+
+.text-primary {
+  color: #f72210 !important;
+}
+
+.border *,
+.border {
+  border: solid 2px black !important;
+}
+
+.border-none {
+  border: none !important;
+}
+
+/* links */
+
+a:hover,
+a:link,
+a:active,
+a:visited {
+  color: currentcolor;
+  text-decoration: none;
```

Comment:
We can use the bootstrap class `text-decoration-none` here

Reply:
Well, `a:visited`, and `a:active`, as well as `a:hover` and `a:link` are pseudo-classes so I think it's not possible to use bootstrap classes on the `html` code.